itest/litd_mode_integrated_test.go

```diff
@@ -618,7 +620,7 @@ func runLNCAuthTest(t *testing.T, hostPort, uiPassword, tlsCertPath string,
 	// endpoint, unless it is explicitly disallowed (we currently don't want
 	// to support creating more sessions through LNC until we have all
 	// macaroon permissions properly set up).
-	resp, err := makeRequest(ctxm, rawLNCConn)
+	resp, err := makeRequest(ctxt, rawLNCConn)
```

Comment:
Shouldn't this still be `ctxm`?

Reply:
as far as I am aware, no. 

`ctxm` contains the litd macaroon. Not the session macaroon. So `ctxm` is used for creating the session yes since that command is using the litd API. The session macaroon is already injected into the request via the authdata which is received during the handshake

Reply:
Aaah, you're totally right.